sdk/communication/communication-network-traversal/test/public/communicationNetworkTraversalClient.spec.ts

```diff
@@ -125,12 +125,20 @@ matrix([[true, false]], async function (useAad) {
     }).timeout(5000);
 
     it("successfully gets a turn credential providing ttl", async function () {
-      const options: GetRelayConfigurationOptions = { ttl: 4000 };
-      const turnCredentialResponse = await client.getRelayConfiguration(options);
-      assert.isNotNull(turnCredentialResponse);
+      const ttl = 4000;
+      const options: GetRelayConfigurationOptions = { ttl: ttl };
 
+      const turnCredentialResponse = await client.getRelayConfiguration(options);
+      const requestedTime = new Date();
```

Comment:
let's move this before getRelayConfiguration call, we don't know how long the network call will take